pkg/icsp/icsp.go

```diff
@@ -0,0 +1,226 @@
+package icsp
+
+import (
+	"context"
+	"fmt"
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/msg"
+	v1alpha1 "github.com/openshift/api/operator/v1alpha1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ICSPBuilder provides struct for the ImageContentSourcePolicy object with connection to the cluster.
+type ICSPBuilder struct {
+	Definition *v1alpha1.ImageContentSourcePolicy
+	Object     *v1alpha1.ImageContentSourcePolicy
+	apiClient  *clients.Settings
+	errorMsg   string
+}
+
+// type AdditionalOptions func(builder *ICSPBuilder) (*ICSPBuilder, error)
+
+// NewICSPBuilder creates a new instance of ICSPBuilder.
+func NewICSPBuilder(apiClient *clients.Settings, name, source string, mirrors []string) *ICSPBuilder {
+	glog.V(100).Infof(
+		"Initializing new ICSPBuilder structure with the following params: \nName: %s\n"+
```

Comment:
Please do not add new lines or capitalize variable names

Reply:
Not sure what you mean here ... 
The line will be to big for the linter 
What you mean by capitalizing the vars here ...

Reply:
Done